log, camera and scoped storage (#19)
owncloud/docs-client-android: modules/ROOT/pages/installation.adoc

It will either update automatically or give you a notification that an upgrade is available.
If you are using an ownCloud Android app from a custom repository, e.g., your employer, then it will update in accordance with their policies.

[NOTE]
====
When upgrading from a release lower than 2.19, the app is using Scoped Storage instead of Shared Storage. See xref:connecting.adoc#physical-location-of-user-files[Physical Location of User Files] for more details.

> The upgrade process takes care transferring all files one by one from Shared Storage to Scoped Storage deleting them from Shared Storage post success for each file. The transition does not require user interaction, runs automatically and is neccessary only once.

Reviewer: ```suggestion
The upgrade process takes care of transferring all files one by one from Shared Storage to Scoped Storage and deleting them from Shared Storage after a successful move of each file. The transition does not require user interaction, runs automatically and only needs to be performed once.
```

====

[WARNING]
====

> Do not power off you device while an upgrade runs the migration process from Shared Storage to Scoped Storage. In case this happens, even accidentially, ownCloud recommends to disconnect all connections and manually delete all Shared Storage data manually. When finished, re-setup all connections which are then automatically using Scoped Storage.

Reviewer: ```suggestion
Do not power off you device while an upgrade runs the migration process from Shared Storage to Scoped Storage. If it happens by accident, ownCloud recommends to disconnect all connections and manually delete all Shared Storage data. When finished, set up all connections once more, which are then automatically using Scoped Storage.
```
